apsi-enrollment-fe/src/app/modules/main/main.routing.ts

```diff
@@ -3,7 +3,8 @@ import { RouterModule, Routes } from '@angular/router';
 import { MainComponent } from './main.component';
 import { AllEventsComponent } from './pages/all-events/all-events.component';
 import { HomeComponent } from './pages/home/home.component';
-import {NewEventComponent} from './pages/new-event/new-event.component';
+import { NewEventComponent } from './pages/new-event/new-event.component';
+import { AllPaymentsComponent } from './pages/all-payments/all-payments.component';
```

Comment:
There is linter rule for imports to be ordered alphabetically. You should use TSLint for that.

Reply:
done